aspdotnet/pyproject.toml

```diff
@@ -53,8 +53,6 @@ include = [
     "/datadog_checks",
     "/tests",
     "/manifest.json",
-    "/requirements-dev.txt",
```

Comment:
No that's a good question. The current `requirements-dev.txt` file only include the ddev package which is now added by default because every integration requires it. The code in the plugin is here: https://github.com/DataDog/integrations-core/blob/master/datadog_checks_dev/datadog_checks/dev/plugin/hatch/environment_collector.py#L39-L43

If other test dependencies are needed, they will now be added directly in the `hatch.toml` file, just like I added the `datadog_checks_tests_helper` here: https://github.com/DataDog/integrations-core/pull/13177/files#diff-807725da3aa4f84d66856fc81dd0e48c1ea3223d31efbd80cf81165672cb0ffbR3-R4